app/models/ems_folder.rb

```diff
@@ -150,6 +150,18 @@ def register_host(host)
     end
   end
 
+  def move_into_folder(vm)
+    raise _("Vm cannot be nil") if vm.nil?
+    vm_mor = vm.ems_ref_obj
+    log_header = "EmsFolder: [#{name}] Vm: id [#{vm.id}], name [#{vm.name}], ems_ref [#{vm_mor}]"
+
+    with_provider_object do |vim_folder|
+      _log.info("#{log_header} Invoking moveIntoFolder ...")
+      vim_folder.send(:moveIntoFolder, vm_mor)
+      _log.info("#{log_header} Invoking moveIntoFolder complete")
```

Comment:
Can you use the more common `Invoking moveIntoFolder...` / `Invoking moveIntoFolder...Complete` ?  That makes it easier to grep for one or both.